test/Interop/Cxx/class/loadable-types-silgen.swift

```diff
@@ -0,0 +1,93 @@
+// RUN: %target-swift-emit-silgen -I %S/Inputs -enable-cxx-interop %s | %FileCheck %s
+
```

Comment:
Please add a short description of the intent of this test. Something like "This test checks that we classify C++ types as loadable and address-only correctly."

Reply:
Done